### DISCUSSION TITLE
Why an empty pull request template, instead of no PR template?

### DISCUSSION THREAD

**Discussion by EliahKagan** [GitoxideLabs/gitoxide#1667] [Q&A] [answered]
The pull request template was introduced in 3d1f45e to ask contributors if they wish for their PR to be featured in a review video. Its content was later removed in c8d6982 (#846) because such videos, though they might sometimes have been possible on request, were no longer routinely made. Since then, [`.github/pull_request_template.md`](https://github.com/GitoxideLabs/gitoxide/blob/main/.github/pull_request_template.md) has been an empty file.

Is there a benefit of it existing as an empty file, or can the file just be removed?

**Comment by Byron** [comment]
I think it *can* be removed and I presume it ended up empty because it contained only one line, which since has been deleted without thinking further.
One argument in favor of keeping it certainly is that it lowers the barrier to adding some content back to an existing file is simpler than having to remember the filename.
It's your call though.

**Reply by EliahKagan** [reply, marked as answer]
> I think it can be removed and I presume it ended up empty because it contained only one line, which since has been deleted without thinking further.

I think it was multiple lines even after the [simplification](https://github.com/GitoxideLabs/gitoxide/pull/494/files) in a9d06a7 (#494).

This may not be significant, but I mention it in case you happened to have been thinking of some other file.

> One argument in favor of keeping it certainly is that it lowers the barrier to adding some content back to an existing file is simpler than having to remember the filename.

In this case, I think that is actually more of an argument against keeping it. There are [multiple ways](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository#adding-a-pull-request-template) to create a pull request template, of which two seem about equally likely for this project: a `.github/pull_request_template.md` file, or multiple files in a `.github/PULL_REQUEST_TEMPLATE` for different kinds of pull requests (much as we have multiple issue templates). Keeping the current empty file may obscure that it is an option to have multiple templates that a developer opening a pull request can select.

> It's your call though.

If further reasons or preferences toward keeping it do not arise, then I may include its removal in a forthcoming PR.

*Edit:* I've included this in #1668.
